app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -96,4 +119,12 @@ export default class MoreConfig extends Component {
 
 MoreConfig.propTypes = {
   onExit: PropTypes.func.isRequired,
+  passedData: PropTypes.objectOf(PropTypes.shape({
+    published: PropTypes.bool.isRequired,
+    allSeries: PropTypes.array.isRequired,
+    canonicalUrl: PropTypes.string.isRequired,
+    series: PropTypes.string.isRequired
+  })).isRequired,
```

Comment:
```suggestion
  }).isRequired,
```

Reply:
yeah. thanks for the stackoverflow link. changes updated @nickytonline